packages/app/src/modules/options/pivotTableConfig.js

```diff
@@ -32,27 +32,44 @@ export default [
         content: [
             {
                 key: 'data-section-1',
-                content: [<ShowDimensionLabels />, <SkipRounding />],
+                /* eslint-disable react/jsx-key */
```

Comment:
Consider just only adding one line of `eslint-disable react/jsx-key`to the top of the file, since it seems to be used everywhere anyway. Cleaner and less duplication 🙂

Reply:
For other sections we are specifying the key here and expect to have it in the component that use these config files, so I went for this approach.

Reply:
Nevermind my comment. Got it. Fixed.